### PR TITLE
add german language

### PR DESCRIPTION
**Pull request by lqs01**

### CODE REVIEW

File: po/de.po [hunk 1]
```diff
@@ -0,0 +1,271 @@
...
+msgid "Devices"
+msgstr "Geräte"
+
+#: src/places.py:92
+msgid "Home"
+msgstr "Persönlicher Ordner"
+
+#: src/places.py:137
+msgid "System"
+msgstr ""
```
Comment (tchx84) [inline po/de.po:176]: It is System in german as well ?

**Review by tchx84 — COMMENTED** [review]
Hey @lqs01 !

Thanks for this, just one question :)

File: po/de.po (hunk 1 above)
Comment (lqs01) [reply po/de.po]: Oh... my fault. Sorry. Changed it to Rechner.

**Comment by tchx84** [comment]
Thanks @lqs01

**Pull request merged**